package.json

```diff
@@ -17,10 +17,11 @@
     "url": "https://github.com/chanzuckerberg/edu-design-system/issues"
   },
   "files": [
-    "/lib"
+    "/lib",
+    "tailwind.config.js"
   ],
   "scripts": {
-    "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn build:styles && yarn copy-to-lib",
+    "build": "yarn build:clean && yarn build:tokens && yarn copy-to-lib && yarn build:js && yarn build:styles",
```

Comment:
to extend this `tailwind.config.js` file in Traject, it needs to reference `lib/tokens` from the built output. That folder gets copied over during `yarn copy-to-lib`, so we now need to move the copy step before `yarn build:styles` happens. Otherwise `yarn build:styles` complains that the folder doesn't exist